tests/monotouch-test/Foundation/DateTest.cs

```diff
@@ -72,8 +71,20 @@ public void Precision32022 ()
 		{
 			var a = NSDate.Now;
 			var b = a.SecondsSinceReferenceDate - ((NSDate) (DateTime) a).SecondsSinceReferenceDate;
-			// ensure decimals are not truncated - but there's an unavoidable loss of precision
-			Assert.AreEqual (b, 0, 0.00001, "1");
+
+			// ensure decimals are not truncated - but there's an unavoidable loss of precision from ns to ms
+			Assert.AreEqual (b, 0, 0.001, "1");
+		}
+
+		[TestCase (1, 1, 1, 1, 1, 1, 1)]
+		[TestCase (199, 1, 1, 1, 1, 1, 1)]
+		[TestCase (2019, 1, 1, 1, 1, 1, 1)]
+		public void DateTimeToNSDate (int y, int m, int d, int h, int min, int s, int ms)
+		{
+			var date = new DateTime (y, m, d, h, min, s, ms);
+			date = DateTime.SpecifyKind (date, DateTimeKind.Utc);
+
+			Assert.AreEqual (date, (DateTime) (NSDate) date, "DateTime -> NSDate -> DateTime conversion");
```

Comment:
Additional test case suggestions?

Reply:
what about using a `DateTimeKind` that isn't UTC? You set the timezone to UTC in your fix, maybe try a test with `Local`?

Reply:
why is this marked as resolved without any accompanying text (or code change) ?

Reply:
nobody suggested any tests so I thought it was clogging up the PR comments... 🤣

Reply:
hmm... @rolfbjarne replies suggested many ?!?

Reply:
this was *before* Rolf's suggestions 😉